iati/core/rulesets.py

```diff
@@ -661,6 +713,13 @@ def __init__(self, context, case):
 
         super(RuleStartsWith, self).__init__(context, case)
 
+    def __str__(self):
+        """A string stating what RuleStartsWith is checking."""
+        if len(self.paths) == 1:
+            return 'Each `{self.paths[0]}` within each `{self.context}` must start with the value present at `{self.start}`.'.format(**locals())
```

Comment:
will make more sense without 'present at'

Reply:
`self.start` is an XPath rather than the prefix itself, so if `present at` is removed then something else needs adding in its place. Any thoughts?